WordPress/gutenberg
# [RNMobile] Apply patch for XCode 12

Fixes: https://github.com/WordPress/gutenberg/issues/25882

## Description

**_This is temporarily solution until all the libs will be updated. Our forks we have to update manually_**

That PR is introducing the patch script on `postinstall` which updates dependency from React to React-Core for all the native libs:

```
-  s.dependency 'React'
+  s.dependency 'React-Core'
```


## How has this been tested?

* run `npm install` or if you have already `node_modules` run `npm postinstall`
* run `npm run native ios`
* app `GutenbergDemo` should build properly





## Screenshots 

## Types of changes

* applying a patch 






## Checklist:
